.github/workflows/integration.yml

```diff
@@ -37,12 +37,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
+        id: global-cache-dir-path
         run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
While we're here, maybe convert these too?

Reply:
I'm doing this in another PR, so I'll leave it for now